src/ui/views/tui_bookmark_menu.rs

```diff
@@ -0,0 +1,127 @@
+use crate::context::AppContext;
```

Comment:
I didn't fully read and understand this file, but it seems to do something similar to `TuiMenu`?
I think it would be better to just create a widget that utilizes `TuiMenu` than to recreate the same logic.